pytorch_lightning/callbacks/gpu_stats_monitor.py

```diff
@@ -187,6 +187,9 @@ def _get_gpu_ids(device_ids: List[int]) -> List[str]:
         return [cuda_visible_devices[device_id].strip() for device_id in device_ids]
 
     def _get_gpu_stats(self, queries: List[str]) -> List[List[float]]:
+        if not queries:
```

Comment:
super n00b question: is `not queries` for truthiness preferred over `if len(queries) == 0` ?